keras/src/saving/file_editor_test.py

```diff
@@ -98,14 +98,16 @@ def test_scalar_weight(self):
         model.add(keras.layers.Dense(1, activation="sigmoid", name="my_dense"))
         model.compile(optimizer="adam", loss="mse", metrics=["mae"])
         model.fit(np.array([[1]]), np.array([[1]]), verbose=0)
```

Comment:
Consider defining these paths as class variables in `setUp()` since they're used multiple times in the test:

```python
def setUp(self):
    super().setUp()
    self.model_fpath = os.path.join(self.get_temp_dir(), "model.keras")
    self.weights_fpath = os.path.join(self.get_temp_dir(), "model.weights.h5")
```

This would avoid repeating the path construction and make the test more maintainable.